modules/apps/sharing/sharing-web/src/main/resources/content/Language.properties

```diff
@@ -10,6 +10,7 @@ edition=Edition
 enter-name-or-email-address=Enter name or email address.
 has-permissions-until=Has Permissions Until
 invite-to-collaborate=Invite to Collaborate
+javax.portlet.title.com_liferay_sharing_web_portlet_ManageCollaboratorsPortlet=Manage Collaborators
```

Comment:
@dantewang @ChiZelin 
I can not find this portlet at all, why we need this line here?

Tina

Reply:
@tinatian This portlet is a Soy portlet registered by a class named `ManageCollaboratorsSoyPortletRegister`.

Reply:
@dantewang I see...
It seems we have different ways to publish a portlet, sometimes we do not have a portlet class with the same name. Thus, we can not add source formatter rule to make *PortletKeys consistent.

A question is: for our portlet title integration test, do we really need to check it for those kind of portlet(without a corresponding real portlet class)? I remembered we added it to avoid some localization problems on UI, are those kind of portlet can be added to UI?


Tina

Reply:
@tinatian Actually many of the portlet we changed in this PR are of `com.liferay.portlet.display-category=category.hidden`. I assume that means the portlets are not for users to add to pages, but embedded somewhere as a part of the UI (mostly in control panel I think).
But I'm not sure whether that makes `javax.portlet.title` key completely unnecessary.